src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -1096,8 +1097,8 @@ export class ChatWidgetService implements IChatWidgetService {
 	private _widgets: ChatWidget[] = [];
 	private _lastFocusedWidget: ChatWidget | undefined = undefined;
 
-	get lastFocusedWidget(): ChatWidget | undefined {
-		return this._lastFocusedWidget;
+	get lastFocusedWidget(): IChatWidget | undefined {
+		return TerminalChatController.activeController?.terminalChatWidget?.inlineChatWidget.chatWidget ?? this._lastFocusedWidget;
```

Comment:
We could hide some of this ugliness in the controller interface: inside `terminalContrib/`:

```ts
TerminalChatController.activeControllers?.chatWidget
```

jsdoc on `chatWidget` vs `terminalChatWidget` would be useful for users of the controller to understand better.